src/buildtool.ts

```diff
@@ -7,7 +7,7 @@ export interface RunBuildOptions {
 }
 
 export function generateCommand(options: RunBuildOptions): string {
-    const pluginsPath = path.join(__dirname,"plugins").replace("'","''");
```

Comment:
Side note: why did we need this in the first place? In case the repo name has a single quote in it?